spec/controllers/idv/verify_info_controller_spec.rb

```diff
@@ -172,6 +173,90 @@
         get :show
       end
     end
+
+    context 'when proofing_device_profiling is enabled' do
+      let(:idv_result) do
+        {
+          context: {
+            stages: {
+              threatmetrix: {
+                transaction_id: 1,
+                review_status: review_status,
+                response_body: {
+                  tmx_summary_reason_code: ['Identity_Negative_History'],
+                },
+              },
+            },
+          },
+          errors: {},
+          exception: nil,
+          success: true,
+        }
+      end
+
+      let(:document_capture_session) do
+        document_capture_session = DocumentCaptureSession.create!(user: user)
+        document_capture_session.create_proofing_session
+        document_capture_session.store_proofing_result(idv_result)
+        document_capture_session
+      end
+
+      let(:expected_failure_reason) { { tmx_summary_reason_code: ['Identity_Negative_History'] } }
+
+      before do
+        controller.
+          idv_session.verify_info_step_document_capture_session_uuid = document_capture_session.uuid
+        allow(IdentityConfig.store).to receive(:proofing_device_profiling).and_return(:enabled)
+      end
+
+      context 'when threatmetrix response is Pass' do
+        let(:review_status) { 'pass' }
+
+        it 'it logs IRS idv_tmx_fraud_check event' do
+          expect(@irs_attempts_api_tracker).to receive(:idv_tmx_fraud_check).with(
+            success: true,
+            failure_reason: nil,
+          )
+          get :show
+        end
+      end
+
+      context 'when threatmetrix response is No Result' do
+        let(:review_status) { 'no_result' }
+
+        it 'it logs IRS idv_tmx_fraud_check event' do
+          expect(@irs_attempts_api_tracker).to receive(:idv_tmx_fraud_check).with(
+            success: false,
+            failure_reason: expected_failure_reason,
+          )
```

Comment:
@jmhooper is this consistent with what we want for Threatmetrix no result? It's treating it as a failure.